node/sc/bridge_manager.go

```diff
@@ -190,54 +189,38 @@ func (bi *BridgeInfo) GetCounterPartToken(token common.Address) common.Address {
 	return cpToken
 }
 
-func (bi *BridgeInfo) GetPendingRequestEvents(start uint64) []*RequestValueTransferEvent {
-	ready := bi.pendingRequestEvent.Ready(start)
+func (bi *BridgeInfo) GetPendingRequestEvents() []*RequestValueTransferEvent {
+	ready := bi.pendingRequestEvent.Pop(maxPendingNonceDiff / 2)
```

Comment:
Question:
Why is it divided by two?

Reply:
I think popping all events every time is weird. So I dividing by 2.
I will tune this value later.